es/lecciones/procesado-basico-de-textos-en-r.md

```diff
@@ -0,0 +1,628 @@
+Barack Obama; 2011; deficit; republicans; democrats; laughter; afghan
+Barack Obama; 2012; afford; deficit; tuition; cannot; doubling
+Barack Obama; 2013; deficit; deserve; stronger; bipartisan; medicare
+Barack Obama; 2014; cory; laughter; decades; diplomacy; invest
+Barack Obama; 2015; laughter; childcare; democrats; rebekah; republicans
+Barack Obama; 2016; laughter; voices; allies; harder; qaida
+```
+
+Como ya habíamos señalado, estos resúmenes temáticos no reemplazan de ninguna manera la lectura atenta de cada documento. Sin embargo, sirven como un resumen de nivel general de cada presidencia. Vemos, por ejemplo, el enfoque inicial en el déficit durante los primeros años de la presidencia de Bill Clinton, su cambio hacia el bipartidismo cuando la Cámara y el Senado se inclinaron hacia los Republicanos en la mitad de los 90 y un cambio hacia una reforma en Medicare al final de su presidencia. Los discursos de George W. Bush se central principalmente en terrorismo, con la excepción del discurso de 2001, ofrecido antes de los ataques terroristas del 11 de Septiembre. Barack Obama volvió a preocuparse por la economía bajo la sombra de la recesión de 2008. La palabra "risa" (laughter) aparece con frecuencia porque se añade a las transcripciones cuando la risa de la audiencia hizo que el emisor tuviera que hacer una pausa.
+
+# Siguientes pasos
+
+En este tutorial breve hemos explorado algunas formas básicas para analizar datos textuales con el lenguaje de programación R. Existen varias direcciones que puedes tomar para adentrarte más en las nuevas técnicas del análisis de texto. Estos son tres ejemplos particularmente interesantes:
+
+* procesar un flujo completo de anotación de procesamiento de lenguajes naturales (NLP) en un texto para extraer características como nombres de entidades, categorías gramaticales y relaciones de dependencia. Estos están disponibles en varios paquetes de R, incluyendo **cleanNLP**, y para varios idiomas[^11].
+* ajustar modelos temáticos (*topic models*) para detectar discursos particulares en el corpus usando paquetes como **mallet**[^12] y **topicmodels**[^13].
+* aplicar técnicas de reducción dimensional para crear gráficos de tendencias estilísticas a lo largo del tiempo o entre múltiples autores. Por ejemplo, el paquete **tsne**[^14] realiza una forma poderosa de reducción dimensional particularmente apta para gráficos detallados.
+
+Existen muchos tutoriales genéricos para estos tres ejemplos, además de documentación detallada de los paquetes[^15]. Esperamos ofrecer tutoriales enfocados en aplicaciones históricas en particular en el futuro.
+
+# Notas
+
+[^1]: Nuestro corpus contiene 236 discursos del Estado de la Unión. Dependiendo de lo que se cuente, este número puede ser ligeramente más alto o más bajo.
+[^2]: Taryn Deward, [“Datos tabulares en R,” Programming Historian](/es/lecciones/datos-tabulares-en-r). Trad. de Jennifer Isasi de 10 de abril de 2018.
```

Comment:
This translation does not appear to be published yet?

Reply:
It's currently under review: https://github.com/programminghistorian/ph-submissions/issues/164

I'm hoping to publish it in the following days. But yes we shouldn't linked it yet...